### PR TITLE
Update remote docs based on experience

### PR DESCRIPTION
**Pull request by pieterlexis**
### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: docs/backends/remote.rst [hunk 1]
```diff
@@ -302,7 +305,7 @@ Query:
 
 .. code-block:: json
 
-    {"method":"getbeforeandafternamesabsolute", "params":{"id":0,"qname":"www.example.com"}}
+    {"method":"getBeforeAndAfterNamesAbsolute", "params":{"id":0,"qname":"www."}}
```
Comment (Habbie) [inline docs/backends/remote.rst:308]: with trailing dot?

Comment (pieterlexis) [reply docs/backends/remote.rst:308]: yes

Comment (Habbie) [reply docs/backends/remote.rst:308]: ew

Comment (pieterlexis) [reply docs/backends/remote.rst:308]: :man_shrugging:

**Review by Habbie — APPROVED** [review]

**Pull request merged**